docs/build-repro.md

```diff
@@ -71,20 +71,12 @@ Workaround:
 # Workaround for https://github.com/moby/buildkit/issues/3180
 ARG SOURCE_DATE_EPOCH
 RUN find $( ls / | grep -E -v "^(dev|mnt|proc|sys)$" ) -newermt "@${SOURCE_DATE_EPOCH}" -writable -xdev | xargs touch --date="@${SOURCE_DATE_EPOCH}" --no-dereference
-```
-
-The `touch` command above is [not effective](https://github.com/moby/buildkit/issues/3309) for mount point directories.
-A workaround is to create mount point directories below `/dev` (tmpfs) so that the mount points will not be included in the image layer.
-
-### Timestamps of whiteouts
-Currently, the `SOURCE_DATE_EPOCH` value is not used for the timestamps of "whiteouts" that are created on removing files.
 
-Workaround:
-```dockerfile
-# Squash the entire stage for resetting the whiteout timestamps.
-# Workaround for https://github.com/moby/buildkit/issues/3168
+# Squashing is needed to apply the touched timestamps across multiple "RUN" instructions.
```

Comment:
> Squashing is needed to apply the touched timestamps across multiple "RUN" instructions.

I'm not sure what this means or how it works. Do you want to say "Squashing is needed so that only file with the defined timestamp from last layer is added to the image"?

> This squashing also addresses non-reproducibility of whiteout timestamps 

This shouldn't be needed in master branch anymore (at least overlay).

Reply:
Updated the description